control-plane/subcommand/gateway-resources/command.go

```diff
@@ -142,6 +154,9 @@ func (c *Command) init() {
 			"gateway container.",
 	)
 
+	c.flags.StringVar(&c.flagMeshGatewayConfigLocation, "meshgw-config-file-location", meshGWConfigFilename,
```

Comment:
This only _happens_ to be for the mesh-gateway only today. In the future, the same config file will contain other gateway types as well, so I'd avoid including meshgw in the flag name.